hazelcast/src/main/java/com/hazelcast/jet/impl/JetClientInstanceImpl.java

```diff
@@ -183,6 +183,8 @@ private void executeJobFromJar(@Nonnull SubmitJobParameters submitJobParameters)
             // Send only job metadata
             logFine(getLogger(), "Submitting JobMetaData for jarPath: %s", jarPath);
             sendJobMetaDataForExecute(jobExecuteCall, submitJobParameters);
+            logFine(getLogger(), "executeJobFromJar for jarPath: %s finished successfully",
```

Comment:
nit: 
```suggestion
            logFine(getLogger(), "Job execution from jar '%s' finished successfully",
```
Reason: more human-friendly (user does not care about method name that much) + added ' to clearly state boundaries of the path

Reply:
Rev : cab8bc17a8721ca01d065a3e36545903bc121fb3